Makefile

```diff
@@ -109,11 +109,15 @@ ocamllex_PROGRAMS = $(addprefix lex/,ocamllex ocamllex.opt)
 
 ocamlyacc_PROGRAM = yacc/ocamlyacc
 
-TOOLS_TO_INSTALL = \
-  ocamldep ocamlprof ocamlcp ocamlmklib ocamlmktop ocamlobjinfo
+# Tools to be compiled to native and bytecode, then installed
+TOOLS_TO_INSTALL_NAT = ocamldep
```

Comment:
I find the convention that `_NAT` means "both in native and bytecode" confusing. Could we:
- not use a prefix in this case, stick to `TOOLS_TO_INSTALL`, `OCAML_PROGRAMS` etc. (the `_BYT` would clearly mean "byte only")
- or use `_BYT_NAT` or `_BYT_AND_NAT`?

Reply:
I feel you're over-thinking it.  I found it pretty clear and visually appealing to split `X` into  `X_NAT` and `X_BYT`.